kratos/mpi/utilities/parallel_fill_communicator.h

```diff
@@ -43,14 +42,19 @@ namespace Kratos
 ///@name Kratos Classes
 ///@{
 
-class ModelPart;
+class ModelPart; // forward declaration
```

Comment:
since we decided to not generally use forward declarations, can you please include it here?

Reply:
This was already here, I think it is a bad idea to abandon the forward declaration

Reply:
It is already included in base class, removing forward declaration